### PR TITLE
Beta: fix minor JS errors

### PR DESCRIPTION
**Pull request by samiff**
#### Changes proposed in this Pull Request:

After PR: https://github.com/Automattic/jetpack/pull/20168

There were a few element not found errors since the plugin layout was adjusted; this PR cleans those errors up.

#### Does this pull request change what data or activity we track or use?

* No.

#### Testing instructions:

* After pulling down changes, the Beta plugin should function as expected still - try the different plugin sections while you have your browser console open.

### CODE REVIEW

File: projects/plugins/beta/src/admin/admin.js [hunk 1]
```diff
@@ -1,4 +1,12 @@
 ( function () {
+	// Avoid running on main plugin selection page.
+	if (
+		'/wp-admin/admin.php?page=jetpack-beta' ===
+		window.location.pathname + window.location.search
```
Comment (anomiex) [inline projects/plugins/beta/src/admin/admin.js:5]: I don't like this check. Look for whether the relevant DOM elements were found instead. Or just don't register it from the PHP in the first place.

File: projects/plugins/beta/src/admin/admin.js [hunk 2]
```diff
@@ -217,6 +233,10 @@
 	 * @param {object} element - DOM Element object.
 	 */
 	function hide( element ) {
+		if ( ! element ) {
```
Comment (anomiex) [inline projects/plugins/beta/src/admin/admin.js:236]: Where is this getting called from without an element?

File: projects/plugins/beta/src/admin/admin.js (hunk 1 above)
Comment (samiff) [reply projects/plugins/beta/src/admin/admin.js]: Thanks for looking Brad, that makes sense. Updated in https://github.com/Automattic/jetpack/pull/20390/commits/cc6cbf64e8d989853b07025af8c9e4930efc41c0

File: projects/plugins/beta/src/admin/admin.js (hunk 2 above)
Comment (samiff) [reply projects/plugins/beta/src/admin/admin.js]: Thinking this was added before I returned out of one of the other functions, but not necessary anymore after https://github.com/Automattic/jetpack/pull/20390/commits/cc6cbf64e8d989853b07025af8c9e4930efc41c0

**Review by anomiex — APPROVED** [review]

**Pull request merged**